CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.47.0
```

Comment:
The version will actually be 0.48.0. We have some weirdness we need to deal with to get the pre-release build to work: 

https://github.com/blindpirate/vscode-pull-request-github/blob/fa65f168e9312aa4f7194fb3834cb5e828a96c82/documentation/releasing.md#L5